.github/workflows/publish.yml

```
@@ -12,15 +12,24 @@ on:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Set Node version
-        uses: actions/setup-node@v3
+        # 🔑 PASO 2: Actualizamos a v4 para soportar OIDC (Trusted Publisher)
+        uses: actions/setup-node@v4
         with:
           node-version-file: ".nvmrc"
           cache: "yarn"
+          registry-url: 'https://registry.npmjs.org/'
+          auth-oidc: true # Habilita la identidad federada
```

actionlint (1.7.12)

[error] input "auth-oidc" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)